lib/percy/common/engine.rb

```diff
@@ -0,0 +1,7 @@
+module Percy
+  module Common
+    class Engine < ::Rails::Engine
```

Comment:
Yeah it only has access to Rails if this file is `require`d from a Rails app. So when we do `require 'percy/common/engine'` inside the Rails app this file will be loaded, but we couldn't ever do this from outside a Rails app.